jest.config.ts

```diff
@@ -5,6 +5,7 @@ const createJestConfig = nextJest({
 });
 
 const config: Config = {
+    preset: "ts-jest",
```

Comment:
jest and cypress can conflict so I made several changes in the configs to prevent that